### ISSUE TITLE
[Request] Variant monochrome (grayscale) icon setting

### ISSUE THREAD

**Issue by GeographicCone** [gorhill/uBlock#1226]
Please consider adding an option in the settings to make uBlock Origin use a monochrome (grayscale) button icon. On Chromium, some other extensions already allow such a choice, which gives more consistency to the UI, especially when there are multiple extension buttons next to one another.

I made monochrome versions of the uBlock Origin icons in 19 px and 38 px, they are attached below along with the SVG source. The colors used are:
* `#727272` for the regular icon and
* `#c4c4c4` when switched off

![icon38_mono](https://cloud.githubusercontent.com/assets/13693118/12317441/4d75a628-bacc-11e5-9df7-a9593405fcc2.png) ![icon38_mono_off](https://cloud.githubusercontent.com/assets/13693118/12317442/4d779b72-bacc-11e5-8cba-4b62848d40d0.png)

To illustrate what it could look like on Chromium (not sure if it would make sense on other platforms):

![ublock_mono_example](https://cloud.githubusercontent.com/assets/13693118/12317226/2657b290-baca-11e5-96f0-e0ccedb1b767.jpg)

_Apparently GitHub refuses to allow a ZIP attachment even though it says it supports them. I'll try again in a moment from a different browser but need to submit the post first as it is._

**Comment by GeographicCone** [comment]
Here are the icons, I created a Gist and posted them into it:
https://gist.github.com/GeographicCone/0c7653f16e529147cc54

![icon38_mono.png](https://gist.github.com/GeographicCone/0c7653f16e529147cc54/raw/27b90b16a6b4cc42c16280e09b4a1e86fe0128ae/icon38_mono.png) ![icon38_mono_off.png](https://gist.github.com/GeographicCone/0c7653f16e529147cc54/raw/27b90b16a6b4cc42c16280e09b4a1e86fe0128ae/icon38_mono_off.png) ![icon19_mono.png](https://gist.github.com/GeographicCone/0c7653f16e529147cc54/raw/27b90b16a6b4cc42c16280e09b4a1e86fe0128ae/icon19_mono.png) ![icon19_mono_off.png]
(https://gist.github.com/GeographicCone/0c7653f16e529147cc54/raw/27b90b16a6b4cc42c16280e09b4a1e86fe0128ae/icon19_mono_off.png)
* [ublock_mono.svg](https://gist.github.com/GeographicCone/0c7653f16e529147cc54/raw/27b90b16a6b4cc42c16280e09b4a1e86fe0128ae/ublock_mono.svg)
* [ublock_mono_off.svg](https://gist.github.com/GeographicCone/0c7653f16e529147cc54/raw/27b90b16a6b4cc42c16280e09b4a1e86fe0128ae/ublock_mono_off.svg)

Thank you for considering this request. I really appreciate all the work you put into uBlock Origin, and uBlock earlier.

**Comment by anewuser** [comment]
Your dark color looks like the light color (off state) of extensions like Lastpass:

![Lastpass icons](https://cloud.githubusercontent.com/assets/6733770/12327137/d88157e6-babb-11e5-9ff7-cb848415deda.png)

Anyway, what you want [can be achieved on Firefox with user CSS](https://www.reddit.com/r/firefox/comments/3ygejp/i_find_the_addon_buttons_too_colourful_and/).

**Comment by GeographicCone** [comment]
> Your dark color looks like the light color (off state) of extensions like Lastpass:

I wouldn't know, I don't use Lastpass. But those colors are in fact taken from another extension that I am familiar with, and they work very well there. That's why I picked them. I am however not attached to these particular values, it was just a proof of concept. As long as it's grayscale, it'd work well for me.

Any particular shades of gray you think would make for a better choice? I can redo the images.

> Anyway, what you want can be achieved on Firefox with user CSS.

I ditched Firefox long time ago (after the Australis fiasco) so I don't even know what their UI looks like this week. On the Chromium platform I feel it would fit very well, making for a more toned-down interface, for those that want it.

"What I want" can also be achieved on Chrome by repacking the extension manually. But it'd be nicer to have a checkbox for this.
